src/shared/api.js

```diff
@@ -1,17 +1,13 @@
 import axios from "axios";
 
-axios.interceptors.request.use((value) => {
-  value.headers = {
-    "Content-Type": "application/json",
-  };
-  return value;
-});
-
-export const api = async (method, section, id, condition, data) => {
+export const api = async (method, endpoint, options = "", data) => {
```

Comment:
the comment was not meant to remove the interceptor entirely; the `json` content header must remain as interceptor header; but the image type should be added to a single request not as interceptor